scripts/automation/alex/alex_pickle.py

```diff
@@ -74,4 +74,4 @@
     print ("Updated the pickle file successfully:\n %s" %pickle.load(f))
 
 
-#print (alex_report_status)
+#print (alex_report_status)
```

Comment:
I guess there is no change in this file. Can we remove this from PR

Reply:
Yes and I removed it.